pkg/controller/import-controller_test.go

```diff
@@ -751,6 +783,29 @@ var _ = Describe("Create Importer Pod", func() {
 		table.Entry("should create pod with file system volume mode and scratchspace", createPvc("testPvc1", "default", map[string]string{AnnEndpoint: testEndPoint, AnnPodPhase: string(corev1.PodPending), AnnImportPod: "podName", AnnPriorityClassName: "p0"}, nil), &scratchPvcName),
 		table.Entry("should create pod with block volume mode and scratchspace", createBlockPvc("testBlockPvc1", "default", map[string]string{AnnEndpoint: testEndPoint, AnnPodPhase: string(corev1.PodPending), AnnImportPod: "podName", AnnPriorityClassName: "p0"}, nil), &scratchPvcName),
 	)
+
+	It("should append current checkpoint name to importer pod", func() {
```

Comment:
Can you add some tests where the pvc name is long, and the snapshot name is long as well. This should trigger the hashing of the name of the pod.

Reply:
I reworked this into a table and added tests for long and short PVC names and checkpoint names.